sandbox/src/routes/index.js

```diff
@@ -4,14 +4,16 @@ const getHealth = require('./getHealth')
 const getPing = require('./getPing')
 const createReferral = require('./createReferral')
 const createReferralAndSendForTriage = require('./createReferralAndSendForTriage')
+const patientServiceSearch = require('./patientServiceSearch')
 
 const routes = [].concat(
-    postCreateProfessionalSession,
-    getStatus,
-    getHealth,
-    getPing,
-    createReferral,
-    createReferralAndSendForTriage
+  postCreateProfessionalSession,
```

Comment:
Is this just the formatting added previously taking effect? Is this formatting implemented as part of the build process or as part of a pre-commit hook?

Reply:
it's on save of the file, it is the formatting I've added earlier on that wasn't there when this was originally added. Things should stabilise now .